SCons/Environment.xml

```diff
@@ -440,7 +460,7 @@ file into an object file.
 </arguments>
 <summary>
 <para>
-Creates an <firstterm>alias</firstterm> target that
+Create an <firstterm>Alias</firstterm> target that
```

Comment:
Should this be Alias entity instead?

Reply:
Maybe the word "Node" is what we want, since that's what is actually made?